os/arch/arm/src/amebad/amebad_flash.c

```diff
@@ -196,41 +166,13 @@ ssize_t amebad_flash_read(size_t addr, void *buf, size_t length)
     flash_t flash;
     int32_t result = 0;
     ssize_t ret = 0;
-    irqstate_t irqs;

```

Comment:
So, regardless aligning, it works for all cases?

Reply:
Yes, I have run the fs_main.c test case and all test cases are passed.